hooks/boot/02-message-helpers.rb

```diff
@@ -0,0 +1,77 @@
+class Kafo::Helpers
+  class << self
+    def success_message
+      say "  <%= color('Success!', :good) %>"
+    end
+
+    def server_success_message(kafo)
+      success_message
+      say "  * <%= color('Katello', :info) %> is running at <%= color('#{kafo.param('foreman', 'foreman_url').value}', :info) %>"
```

Comment:
Would it make sense to avoid the use of Katello here? Either the application or the interface could work.

Reply:
You mean saying `Foreman` or `Application` ?

Reply:
If you avoid any branded words and use generic descriptions like ```Application``` then there's no difference between upstream & downstream which makes life easier.

Reply:
It does, but there are some instances where that is unavoidable such as the commands or package names. I can make as much as possible generic but it won't prevent this situation.

Reply:
That would be preferable. With commands I'd prefer a git-style application where you can brand the main binary (foreman-maintain -> satellite-maintain) and subcommands are unbranded. You are correct that in some cases it will be unavoidable but let's aim at minimizing that.

Reply:
Alright, I'll change as much as I can although I will lastly state that I think moving to generics while making this easier, reduces the user experience.

Reply:
The more I look at it, this is about the only line we save anything on, and thus its cleaner to just change this to 'Foreman'  and leave the rest as is at the risk of just being overly generic and ambigous.

Reply:
I think the Foreman is fine.